### PR TITLE
add integration tests for network module

### LINKED ISSUE #280: Integration tests for lib.analysis.network module
The following functions of the module need integration tests.
1. lib.analysis.network.degree_node_number_csv()
1. lib.analysis.network.identify_hubs_and_experts()
1. lib.analysis.network.message_number_graph()
1. lib.analysis.network.message_number_bins_csv()
1. lib.analysis.network.message_time_graph()

### PR DESCRIPTION
**Pull request by kaushik-rohit**
Integration tests for:-
   1) lib.analysis.network.degree_node_number_csv()
   2) lib.analysis.network.identify_hubs_and_experts()
   3) lib.analysis.network.message_number_graph()
   4) lib.analysis.network.message_number_bins_csv()
   5) lib.analysis.network.message_time_graph()

Issue Link: https://github.com/prasadtalasila/IRCLogParser/issues/280

## What? Why?
Fix #280 

## Checks
- [x] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Make sure you have added required documentation
- [x] If you have refactored a code, make sure that you have compared the outputs pre and post refactoring. Add both pre and post refaoring screenshots below if the output is visual

## Notify reviewers
@kaushik-rohit
@prasadtalasila

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@kaushik-rohit all the expected variables need to have `expected_` as a prefix. Please enforce the same in the unit tests of network.py module as well.

**Comment by kaushik-rohit** [comment]
@prasadtalasila The work is complete now. You can review/merge it.

File: test/integration-test/test_lib/test_analysis/test_network.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    @mock.patch('lib.config.MAX_EXPECTED_DIFF_NICKS', 5000)
+    @mock.patch('lib.config.THRESHOLD_MESSAGE_NUMBER_GRAPH', 0)
+    @mock.patch('lib.config.MINIMUM_NICK_LENGTH', 3)
+    @mock.patch('lib.config.DEBUGGER', True)
+    def test_message_number_graph_day_analysis(self):
+
+        capturedOutput = StringIO.StringIO()
+        sys.stdout = capturedOutput
+
+        ret = network.message_number_graph(self.log_data, self.nicks, self.nick_same_list, DAY_BY_DAY_ANALYSIS=True)
```
Comment (prasadtalasila) [inline test/integration-test/test_lib/test_analysis/test_network.py]: it is better to name the `ret` as `graph`.

File: test/integration-test/test_lib/test_analysis/test_network.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        self.assertTrue(nx.is_isomorphic(ret[0][0], expected_graph_list[0][0]))
+        self.assertTrue(nx.is_isomorphic(ret[1][0], expected_graph_list[1][0]))
+
+    @mock.patch('lib.config.MAX_EXPECTED_DIFF_NICKS', 5000)
+    def test_message_time_graph(self):
+
+        capturedOutput = StringIO.StringIO()
+        sys.stdout = capturedOutput
+
+        ret = network.message_time_graph(self.log_data, self.nicks, self.nick_same_list, DAY_BY_DAY_ANALYSIS=False)
```
Comment (prasadtalasila) [inline test/integration-test/test_lib/test_analysis/test_network.py]: It is better to name this variable as `msg_time_aggr_graph`

**Review by prasadtalasila — CHANGES_REQUESTED** [review]
@kaushik-rohit please make minor changes to variable names.

**Review by prasadtalasila — APPROVED** [review]

**Pull request merged**